src/main/java/io/openliberty/website/dheclient/DHEBuildParser.java

```diff
@@ -149,6 +152,13 @@ private synchronized void clearScheduledUpdate() {
 		scheduledUpdate = null;
 	}
 
+	private <T> Future<T> submit(Callable<T> c) {
+//		if (exec == null) {
```

Comment:
Will this be potentially used later?

Reply:
no. I had at one point had that for test, but it was causing it to be harder to debug in the server so I ended up moving it to the test constructor and didn't remove this code. Fixed now though.